assets/xml/objects/object_bal.xml

```diff
@@ -1,81 +1,93 @@
-        <Limb Name="object_bal_Standardlimb_00A5C0" Type="Standard" EnumName="OBJECT_BAL_LIMB_15" Offset="0xA5C0" />
-        <Limb Name="object_bal_Standardlimb_00A5CC" Type="Standard" EnumName="OBJECT_BAL_LIMB_16" Offset="0xA5CC" />
-        <Limb Name="object_bal_Standardlimb_00A5D8" Type="Standard" EnumName="OBJECT_BAL_LIMB_17" Offset="0xA5D8" />
-        <Limb Name="object_bal_Standardlimb_00A5E4" Type="Standard" EnumName="OBJECT_BAL_LIMB_18" Offset="0xA5E4" />
-        <Limb Name="object_bal_Standardlimb_00A5F0" Type="Standard" EnumName="OBJECT_BAL_LIMB_19" Offset="0xA5F0" />
-        <Limb Name="object_bal_Standardlimb_00A5FC" Type="Standard" EnumName="OBJECT_BAL_LIMB_1A" Offset="0xA5FC" />
-        <Limb Name="object_bal_Standardlimb_00A608" Type="Standard" EnumName="OBJECT_BAL_LIMB_1B" Offset="0xA608" />
-        <Limb Name="object_bal_Standardlimb_00A614" Type="Standard" EnumName="OBJECT_BAL_LIMB_1C" Offset="0xA614" />
-        <Limb Name="object_bal_Standardlimb_00A620" Type="Standard" EnumName="OBJECT_BAL_LIMB_1D" Offset="0xA620" />
-        <Limb Name="object_bal_Standardlimb_00A62C" Type="Standard" EnumName="OBJECT_BAL_LIMB_1E" Offset="0xA62C" />
-        <Limb Name="object_bal_Standardlimb_00A638" Type="Standard" EnumName="OBJECT_BAL_LIMB_1F" Offset="0xA638" />
-        <Limb Name="object_bal_Standardlimb_00A644" Type="Standard" EnumName="OBJECT_BAL_LIMB_20" Offset="0xA644" />
-        <Skeleton Name="object_bal_Skel_00A6D0" Type="Flex" LimbType="Standard" LimbNone="OBJECT_BAL_LIMB_NONE" LimbMax="OBJECT_BAL_LIMB_MAX" EnumName="ObjectBalLimb" Offset="0xA6D0" />
-        <Animation Name="object_bal_Anim_00A7DC" Offset="0xA7DC" /> <!-- Original name is "bal_put" -->
-        <Animation Name="object_bal_Anim_00B1E8" Offset="0xB1E8" /> <!-- Original name is "bal_talk" -->
-        <Animation Name="object_bal_Anim_00B604" Offset="0xB604" /> <!-- Original name is "bal_talk2" -->
-        <Animation Name="object_bal_Anim_00C498" Offset="0xC498" /> <!-- Original name is "bal_talk3" -->
-        <Animation Name="object_bal_Anim_00C8D8" Offset="0xC8D8" /> <!-- Original name is "bal_talk4" -->
-        <Animation Name="object_bal_Anim_00CB78" Offset="0xCB78" /> <!-- Original name is "bal_tereru" ("to be shy; to be bashful") -->
-        <Animation Name="object_bal_Anim_00D530" Offset="0xD530" /> <!-- Original name is "bal_wait" -->
+
+    <!-- Animations 1 -->
+        <Animation Name="gTingleFloatIdleAnim" Offset="0x5FC" /> <!-- Original name is "bal_fly"-->
+        <Animation Name="gTingleFallAnim" Offset="0x840" /> <!-- Original name is "bal_fly2"-->
+        <Animation Name="gTingleSpinAnim" Offset="0xC78" /> <!-- Original name is "bal_kaiten" ("rotation; revolution; turn; spin​") -->
+        <Animation Name="gTingleThrowConfettiAnim" Offset="0x1804" /> <!-- Original name is "bal_kami" ("paper​")-->
+
+    <!-- Model Pieces -->
+        <DList Name="gTingleBriefsDL" Offset="0x4000" />
```

Comment:
I'd prefer if this kept the name of the associated limb intact (e.g., it would be called `gTinglePelvisDL`). The reason being that you could texture this or color this in various ways to make it *not* be his briefs, but you can't make it *not* be his pelvis without editing the model.